acquia_lift.admin.unibar.inc

```diff
@@ -981,7 +479,7 @@ function acquia_lift_element_variation_delete_form($form, &$form_state, $osid, $
 }
 
 /**
- * Submit handler for acquia_lift_page_variation_delete_form().
+ * Submit handler for acquia_lift_element_variation_delete_form().
```

Comment:
I am actually quite amazed the amount of copy pasta that I am *not* seeing in this code base. You are Kat are pretty neat. Gotta step up my game now haha.